client/components/token-field/index.jsx

```diff
@@ -171,6 +180,10 @@ class TokenField extends PureComponent {
 			value: this.state.incompleteTokenValue,
 		};
 
+		if ( props.autoCorrect === 'off' ) {
+			props.spellCheck = 'false';
```

Comment:
I wasn't familiar with [`spellcheck`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-spellcheck):

> `spellcheck` HTML5
> Setting the value of this attribute to `true` indicates that the element needs to have its spelling and grammar checked. The value `default` indicates that the element is to act according to a default behavior, possibly based on the parent element's own `spellcheck` value. The value `false` indicates that the element should not be checked.

My read is that this is similar but not the same as [`autocorrect`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-autocorrect):

> `autocorrect` 
> This is a non-standard attribute supported by Safari that is used to control whether autocorrection should be enabled when the user is entering/editing the text value of the `<input>`. Possible attribute values are:
>
> * `on`: Enable autocorrection.
> * `off`: Disable autocorrection.
> 
> [`autocorrect` documentation](https://developer.apple.com/library/safari/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/Attributes.html#//apple_ref/doc/uid/TP40008058-autocorrect) in the Safari HTML Reference.

I wonder whether disabling `autocorrect` should imply disabling `spellcheck`. What was the motivation for adding `spellCheck` here?

Reply:
I think this is a good question. Is it a better idea to specify those two separately? Although in a `TokenField`, I doubt it will be particularly useful.

Reply:
I combined them in this manner as turning off `autoCorrect` on its own apparently has no effect on Safari when `spellcheck` isn't turned off as well ([#](https://stackoverflow.com/a/35514029)). In essence it's a way to guard developers from a browser quirk. 

What do you think?

Reply:
I've just done some quick testing.

It seems that newer desktop safari (11) respects the standard `spellcheck="false"` without needing `autocorrect`.

However, testing safari on iOS 10.3.3 does seem to require both.

Given that `spellcheck` is the standard, I think I'd prefer to invert this, setting `autoCorrect` based on `spellCheck`, and pass the `spellCheck` prop.

Reply:
Hmmm — yeah, I understand your perspective. 

But ... we don't really care about spellcheck here, though, do we? At least it's not the worst of the two. It's much more annoying to have a username changed by auto-correct than having it underlined. That's why we were starting from auto-correct, not spellcheck. 

Right now I'm thinking that maybe leaving the decision to the developer as you suggested previously might be the best choice, after all. 😬

Reply:
> leaving the decision to the developer
👍